_posts/2017-08-16-Refactoring-with-ReactJS-testing.md

```diff
@@ -27,38 +27,39 @@ test_tools:
 
 *Note: This post is part of a series, starting with [Refactoring a Legacy Application with React - Introduction]({% post_url 2017-08-14-Refactoring-with-reactJS-introduction %})*
 
-In this post we are going to cover the essential part that makes move to React as a front-end framework so highly desirable. We are talking of course about testing.
+In this post we are going to cover the essential part that makes the move to React as a front-end framework so highly desirable. We are talking of course about testing.
 
 # Topics we cover
   * Tool set with short description of game pieces and choices
   * Test utility configuration
   * Homegrown helper utilities
   * Basic DOM rendering tests
-  
+
 ## Tool set
-We start of with actual tools that we are using in our legacy stack. Our organization had prior in-depth front-end work 
-done in our legacy application. We have had a set of tools already in use and we decided to continue the trend and reuse
-things that have been deployed. That being said you might pick a different set of tools especially if you are starting
-with testing completely anew.<br />
+We start of with actual tools that we are using in our legacy stack. Our organization had prior in-depth front-end work done in our legacy application. We have a set of tools already in use and we decided to continue the trend and reuse things that have been deployed. That being said, you might pick a different set of tools especially if you are starting with testing completely anew.
+
 **So here is our tool set:**
 {% assign toolz = page.test_tools %}
-<div class="container">
-    {% for tool in toolz %}
-        <div class="row">
-        <div class="col-md-1 vcenter">
-        <a href="{{tool.tool_url}}"><img class="tool-icon" src="{{ tool.image_path }}"  /></a>
-        </div><div class="col-md-10 vcenter">
-        <a href="{{tool.tool_url}}"><b>{{ tool.tool_name }}</b></a> - {{ tool.tool_description }}
-        </div></div>
-    {% endfor %}
+<div>
+  {% for tool in toolz %}
+      <div class="media">
```

Comment:
Changed the tools layout to a Bootstrap media object. It will be a bit more responsive.